packages/api/src/command/medical/document/generate-ccd.ts

```diff
@@ -0,0 +1,448 @@
+import { Address as FhirAddress, Organization } from "@medplum/fhirtypes";
+import { AddressStrict } from "@metriport/core/domain/location-address";
+import { Address } from "@metriport/core/src/domain/address";
+import { Contact } from "@metriport/core/src/domain/contact";
+import { metriportOrganization } from "@metriport/shared/common/metriport-organization";
+import { OrganizationModel } from "../../../models/medical/organization";
+import { getOrganizationOrFail } from "../organization/get-organization";
+import { getPatientOrFail } from "../patient/get-patient";
+import { Config } from "../../../shared/config";
+
+const metriportOid = Config.getSystemRootOID();
+export async function generateEmptyCcd({
+  patientId,
+  cxId,
+}: {
+  patientId: string;
+  cxId: string;
+}): Promise<string> {
+  const organization = await getOrganizationOrFail({ cxId });
+  const patient = await getPatientOrFail({ cxId, id: patientId });
+  const data = patient.data;
+  const address = data.address;
+  const addresses = buildAddresses(address);
+  const patientTelecom = buildTelecom(data.contact);
+  const author = buildAuthor(organization);
+  const custodian = buildCustodian(metriportOrganization);
+  const structuredBody = buildStructuredBody();
+  const currentTime = getCurrentDate();
+
+  return `<ClinicalDocument xmlns="urn:hl7-org:v3" xmlns:sdtc="urn:hl7-org:sdtc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" moodCode="EVN">
```

Comment:
I know FHIR to CDA wasn't merged yet, which afaik has functions and a framework for generating CDA right? 

Maybe we can still use fast-xml-parser as a library so we stay away from string replacing and work with nice json instead? 

Wdyt? Alternative is to refactor later

Reply:
I don't see much benefit in making the switch now. The current code gets the job done. Maybe once the FHIR-to-CDA PRs are shipped

Reply:
Can we at least ticket it then? For here and for [here](https://github.com/metriport/metriport/blob/9ca8186ca79ac2599d6bd3136e6e500cce4339ed/packages/core/src/external/carequality/dq/create-metadata-xml.ts#L4) to migrate to fast-xml-parser's builder?